Scripts/Python/xOptionsMenu.py

```diff
@@ -702,80 +698,15 @@ def OnGUINotify(self,id,control,event):
                 kmID = control.getTagID()
                 if kmID == kKMOkBtn:
                     KeyMapDlg.dialog.hide()
-                elif kmID in gKM1ControlCodesRow1.keys():
-                    NewKeyMapString = ""
-                    # get the new keys and bind
-                    km = ptKeyMap()
-                    cCode,spFlag,mpFlag = gKM1ControlCodesRow1[kmID]
-                    if isinstance(cCode, str):
-                        key1 = km.convertVKeyToChar(control.getLastKeyCaptured(),control.getLastModifiersCaptured())
-                        km.bindKeyToConsoleCommand(key1,cCode)
-                        KeyMapString = self.getChronicleVar("KeyMap")
-                        KeyMapArray = KeyMapString.split()
-                        KeyMapArray[(kmID-300)] = key1
-                        for key in KeyMapArray:
-                            NewKeyMapString += key + " "
-                        self.setNewChronicleVar("KeyMap", NewKeyMapString.rstrip())
-                    elif cCode is not None:
-                        otherID = kmID + 100
-                        otherField = ptGUIControlEditBox(KeyMapDlg.dialog.getControlFromTag(otherID))
-                        key1 = km.convertVKeyToChar(control.getLastKeyCaptured(),control.getLastModifiersCaptured())
-                        key2 = km.convertVKeyToChar(otherField.getLastKeyCaptured(),otherField.getLastModifiersCaptured())
-                        controlStr = km.convertControlCodeToString(cCode)
-                        km.bindKey(key1,key2,controlStr)
-                        KeyMapString = self.getChronicleVar("KeyMap")
-                        KeyMapArray = KeyMapString.split()
-                        KeyMapArray[(kmID-300)] = key1 + "$" + key2 + " "
-                        for key in KeyMapArray:
-                            NewKeyMapString += key + " "
-                        self.setNewChronicleVar("KeyMap", NewKeyMapString.rstrip())
-                    # lose the focus when done
-                    KeyMapDlg.dialog.noFocus()
-                    # force writing the keymap
-                    km.writeKeyMap()
```

Comment:
The new version no longer calls `km.writeKeyMap()` anywhere. I assume this is intentional?

Perhaps we should just remove `ptKeyMap.writeKeyMap` and the underlying `plInputInterfaceMgr::WriteKeyMap()`. AFACIT that code did nothing useful anyway. It tries to write a file init/keyboard.fni, but MOULa installs normally don't have an init folder, so that always fails silently.

Reply:
Right, I intentionally removed this call because it has no effect. I didn't want to remove the `writeKeyMap` code in this PR, though.